### PR TITLE
fix: bring back #22

### PR DESCRIPTION
**Pull request by Ravencentric**
https://github.com/ThaUnknown/releases-moe/pull/22 was reverted in an attempt to fix builds but we found out https://github.com/ThaUnknown/releases-moe/pull/22 had nothing to do with it

### CODE REVIEW

File: sk/src/app.pcss [hunk 1]
```diff
@@ -4,67 +4,48 @@
  
 @layer base {
   :root {
-    --background: 0 0% 100%;
-    --foreground: 240 10% 3.9%;
-
```
Comment (ThaUnknown) [inline sk/src/app.pcss:9]: make the spacing the same at least, easier to diff :p

Comment (Ravencentric) [reply sk/src/app.pcss:9]: fixed spacing

**Comment by ThaUnknown** [comment]
holy what happened to the order of these, are any values missing or are they just reordered?

**Comment by Ravencentric** [comment]
1 value added: https://github.com/ThaUnknown/releases-moe/pull/23/files#diff-46ba280e81e456eb66a1aea611226faad1c3ebb5932b4cb96b8cc2337fe244a2R69

I fixed the ordering too

**Review by ThaUnknown — APPROVED** [review]

**Pull request merged**